sorting/Bogo_Sort.cpp

```diff
@@ -53,7 +66,7 @@ std::vector <int> randomized_bogosort(std::vector <int> arr)
 //Driver Code
 int main()
 {
-    std::vector <int> arr = {3, 7, 10, 4, 1}; // Defining array which we want to sort
+    std::array <int, 5> arr = {3, 7, 10, 4, 1}; // Defining array which we want to sort
```

Comment:
can you please branch it into another function `void test()` and create a couple of arrays and sort them? 
You can use `std::rand()` to create random arrays and check with `std::is_sorted`.